### DISCUSSION TITLE
Core team members

### DISCUSSION THREAD

**Discussion by bgrand-ch** [GrapesJS/grapesjs#5238] [Ideas]
Hello @artf,

Do you have already thinking about a core team members program to enroll motivated developers to join "officially" this project and assist you?

- Dedicated to answer most questions in "Discussions".
- Check pull requests and validate them.
- Assist companies that have explicitly made a recurring donation.
- Create a roadmap and assign certain tasks to team members.
- Maintain documentation and add usage examples.
- ...

🤔

**Comment by gxanshu** [comment]
I wanna join too, working with greapes from a long time now

**Comment by ronaldohoch** [comment]
Me too!

**Comment by artf** [comment]
Thanks @bgrand-ch that's a really good question.

Of course, I'd love to have more people involved with the project, but you know, usually, people are motivated and willing to help until there is an open interest that drives that motivation, once the interest is gone (eg. the project they were working on is "done" or they simply switched on something else) people disappears and obviously no one can blame them.

So currently, I see it more like an open space, where everyone is free to decide what/where they're willing to help and no one is forced on the specific context, but obviously, if anybody is open to focusing on something specific I won't be against that.

From your specific points I'd say those are currently open for everyone:
* Dedicated to answer most questions in "Discussions".
* Check pull requests and validate them.
* Maintain documentation and add usage examples.
* Assist companies that have explicitly made a recurring donation.


What is actually missing right now is definitely the proper Roadmap (the [old one](https://github.com/GrapesJS/grapesjs/issues/74) is way too outdated). 
I'll try to find time to create one soon, probably it would also be easier to fund specific features/fixes.
